aff4/libaff4-c.cc

```diff
@@ -153,6 +153,7 @@ AFF4_Handle* AFF4_open(const char* filename, AFF4_Message** msg) {
     };
 
     h->volumes.AddVolume(std::move(zip));
+    h->volumes.AddSearchPath(filename);
```

Comment:
search path is not related to the volume group (volumes have no concept of paths). This should be a separate helper object or function.